File: javA/basics/Triangle.java
package basics;
public class Triangle {

    public static double getArea( double a, double b, double c) {

        if (a+b>c && b+c>a && a+c>b && a > 0 && b > 0 && c > 0) {
            double halfCircum = (a+b+c)/2;

            double area = ( Math.sqrt( halfCircum * (halfCircum - a) * (halfCircum - b) * (halfCircum - c) ) );

            return area;
        }

        else {
            return 0;
        }

    }

    public static void main(String[] args) {

        double a = Double.parseDouble(args[0]), b = Double.parseDouble(args[1]), c = Double.parseDouble(args[2]);

        double area = getArea(a,b,c);

        System.out.println(area);
    }
    
}

File: pYthon/practice/week1/triangle.py
import math


def area(a, b, c):
    s = (a + b + c) / 2
    return math.sqrt(s * (s - a) * (s - b) * (s - c))


def is_triangle(a, b, c):
    if (a + b > c and b + c > a and a + c > b):
        return True

    else:
        return False



def main():
    a = float(input())
    b = float(input())
    c = float(input())

    if (is_triangle(a, b, c) == True):
        s = area(a, b, c)
        c = a + b + c

        print(f"S =  {s} C =  {c}")

    else:
        print("INVALID")

main()
